functests/06-test-deploy-node-labeller.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+[ -z ${KV_NAMESPACE} ] && exit 127
+
+RET=1
+TEST_NS="${KV_NAMESPACE}"
+
+oc create -n ${TEST_NS} -f "${SCRIPTPATH}/node-labeller-unversioned-cr.yaml" || exit 2
+# TODO: SSP-operator needs to improve its feedback mechanism
+sleep 10s
+for idx in $( seq 1 30); do
```

Comment:
I am not sure if 70 seconds is enough for node labeller, sometimes it is really slow to download all images. Maybe i would do something like this: https://github.com/kubevirt/node-labeller/pull/12/files#diff-80e9ee6c88bb79025632ec61e99d4feeR18
and add there some timeout counter.

Reply:
Thanks for the heads up. My first thought is we should try to solve this from a different perspective, pre-downloading all the needed images. I'll try this direction first.

Reply:
While I work on the approch I thought, I added code to implement Karel's suggestion.